docs: Document state=running for compute_instance (#203)
exoscale/terraform-provider-exoscale: examples/README.md

An example showing how the [Private networks](https://www.exoscale.com/syslog/introducing-multiple-private-networks/)
API support can be used to create a private network between compute instances.

## [Managed Private Network](managed-private-network)

> The vanilla private networks having no configuration this shows how to enable

Reviewer: ```suggestion
Vanilla private networks are unconfigured by default; the following shows how to enable
```
